libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1106,6 +1109,27 @@ bool AP_AHRS_DCM::airspeed_estimate(uint8_t airspeed_index, float &airspeed_ret)
     return true;
 }
 
+bool AP_AHRS_DCM::get_unconstrained_airspeed_estimate(uint8_t airspeed_index, float &airspeed_ret) const
+{
+#if AP_AIRSPEED_ENABLED
+    if (airspeed_sensor_enabled(airspeed_index)) {
+        airspeed_ret = AP::airspeed()->get_airspeed(airspeed_index);
+        return true;
+    }
+#endif
+
+    if (AP::ahrs().get_wind_estimation_enabled() && have_gps()) {
+        // estimate it via GPS speed and wind
```

Comment:
```suggestion
        // estimated via GPS speed and wind
```

... this is actually somewhat bogus, we should be keeping a boolean or timestamp around rather than checking `get_wind_estimation_enabled() && have-gps()`.  Those two could go true while `_last_airspeed` is still invalid (if not with the current codepaths then via some future well-meaning change.

However, pre-existing.